providers/supermicro/supermicro.go

```diff
@@ -98,6 +98,7 @@ func (s *supermicro) GetInventory(ctx context.Context, options ...actions.Option
 			firmware.MakeOutputPath(),
 			firmware.TraceExecution(trace),
 		),
+		UEFIVarsCollector: &utils.UEFIVariableCollector{},
```

Comment:
Is there a reason we are only doing UEFIVars collection on SMC? Could it be added to other providers too or added somewhere more generic? NOT A BLOCKER FOR LANDING!

Reply:
its by default collected on the rest